pkg/sessions/session_store_test.go

```diff
@@ -239,6 +277,28 @@ var _ = Describe("NewSessionStore", func() {
 		})
 	})
 
+	Context("with type 'redis'", func() {
+		BeforeEach(func() {
+			mr, err := miniredis.Run()
+			if err != nil {
+				panic(err)
```

Comment:
This `if` clause should be an `Expect(err).ToNot(HaveOccurred())`, which will fail the test at this point if the err is not nil